axonserver/src/main/java/io/axoniq/axonserver/component/version/Version.java

```diff
@@ -0,0 +1,72 @@
+package io.axoniq.axonserver.component.version;
+
+import static java.lang.Integer.parseInt;
+
+/**
+ * Represents an artifact version.
+ *
+ * @author Sara Pellegrini
+ * @since 4.2.1
```

Comment:
I see a mix of `since 4.2.1`, `since 4.2.2` and `since 4.2.3`. Is this intentional?

Reply:
No, it was not intentional, I guess I created classes before to perform the release. Now I've fixed them.